spec/services/arcgis_api/geocoder_spec.rb

```diff
@@ -33,4 +33,29 @@
       )
     end
   end
+
+  describe '#find_address_candidates' do
```

Comment:
Should we include some error cases here? Maybe as `shared_examples` or helper method if we're expecting it to behave the same across the two API calls.